include/bluetooth/services/ots.h

```diff
@@ -31,6 +31,9 @@ extern "C" {
 /** @brief Size of OTS object ID (in bytes). */
 #define BT_OTS_OBJ_ID_SIZE 6
 
+/** @brief Mask for OTS object IDs, preserving the 48 bits */
+#define BT_OTS_OBJ_ID_MASK 0x0000FFFFFFFFFFFFUL
```

Comment:
I wonder if it would be more clear if it was 
```suggestion
#define BT_OTS_OBJ_ID_MASK BIT_MASK(48)
```

Reply:
Certainly easier to read than sitting there counting four zeros and twelve "F"s.
Changed.